core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -211,6 +211,31 @@ private[spark] class AppStatusListener(
           update(rdd, now)
         }
       }
+      // remove partition information
+      liveRDDs.values.foreach { rdd =>
+        rdd.getPartitions.values
+          .filter(_.executors.contains(event.executorId)).foreach { partition =>
```

Comment:
Indentation is weird - the blocks fuse together. Use:

```
 rdd.getPartitions.values
  .filter(_.executors.contains(event.executorId))
  .foreach { partition =>
    // code
  }
```

Reply:
corrected